CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
+ * Support for validation, completion and document link for `#include` and `#insert` sections in Qute templates. See [#438](https://github.com/redhat-developeredhat-developer/quarkus-lsr/quarkus-ls/issues/438).
+ * "Show references" CodeLens for `#insert` section in Qute templates. See [#595](https://github.com/redhat-developer/vscode-quarkus/pull/595).
+ * Support surround with section/comments command for Qute templates. See [#581](https://github.com/redhat-developer/vscode-quarkus/issues/581).
+ * Provide syntax coloration for #fragment. See [#570](https://github.com/redhat-developer/vscode-quarkus/pull/570).
+
+### Bug Fixes
+
+ * Treat `CompletionStage` and `Uni` objects as their resolved type for validation and completion in Qute templates. See [redhat-developer/quarkus-ls#826](https://github.com/redhat-developer/quarkus-ls/issues/826).
+ * `{#if}` sections in Qute templates of the form `{#if !inject:beanName}{/if}` are marked as invalid. See [redhat-developer/quarkus-ls#828](https://github.com/redhat-developer/quarkus-ls/issues/828).
+ * Quick fix to generate a `@TemplateExtension` class from a Qute template might place the class in the wrong folder. See [redhat-developer/quarkus-ls#831](https://github.com/redhat-developer/quarkus-ls/issues/831).
+ * "Create project" fails when there are no folders and one java file open. See [#559](https://github.com/redhat-developer/vscode-quarkus/issues/559).
+ * Fix Quarkus debug configuration after maven wrapper is deleted. See [#572](https://github.com/redhat-developer/vscode-quarkus/issues/572).
+ * qute-ls isn't stopped by vscode-quarkus. See [#576](https://github.com/redhat-developer/vscode-quarkus/issues/576).
+ * Prevent infinite loop when going to the definition of a user tag. See [#579](https://github.com/redhat-developer/vscode-quarkus/issues/579).
+ * Prevent popups by reading files without triggering `onDidOpenTextDocument`. See [#584](https://github.com/redhat-developer/vscode-quarkus/pull/584).
+ * Prevent errors while vscode-java is still starting. See [#586](https://github.com/redhat-developer/vscode-quarkus/issues/586).
+ * Incomplete `{#if}` section with operator may generate `ClassCastException`. See [redhat-developer/quarkus-ls#816](https://github.com/redhat-developer/quarkus-ls/issues/816).
+ * Use quarkus-ls's Qute parser for syntax validation in order to get consistent diagnostic positions and ranges. See [redhat-developer/quarkus-ls#812](https://github.com/redhat-developer/quarkus-ls/issues/812).
+ * Completion in Qute template should generate only start section when an empty end section is found. See [redhat-developer/quarkus-ls#805](https://github.com/redhat-developer/quarkus-ls/issues/805).
+ * Quarkus 3 support. See [redhat-developer/quarkus-ls#779](https://github.com/redhat-developer/quarkus-ls/issues/779).
```

Comment:
Would this be considered a bug fix?

Reply:
I listed it as a bug under quarkus-ls, but I agree it's an important feature for this release, so I'll move it to features here.